mavros/src/lib/uas_frame_conversions.cpp

```diff
@@ -21,24 +21,75 @@ using namespace mavros;
 // Eigen based functions
 
 //! +PI rotation around X (Roll) axis give us ROS or FCU representation
-static const Eigen::Quaterniond FRAME_ROTATE_Q = UAS::quaternion_from_rpy(M_PI, 0.0, 0.0);
+static Eigen::Quaterniond FRAME_ROTATE_Q;
 
 //! Transform for vector3
-static const Eigen::Transform<double, 3, Eigen::Affine> FRAME_TRANSFORM_VECTOR3(FRAME_ROTATE_Q);
+//static const Eigen::Transform<double, 3, Eigen::Affine> FRAME_TRANSFORM_VECTOR3(FRAME_ROTATE_Q);
```

Comment:
It is ok for research, but later i think better to make two consts: `FRAME_ROTATE_LOCAL_Q`, `FRAME_ROTATE_PLATFORM_Q`.

Also `platform`? Not `base`?